relay-server/src/services/store.rs

```diff
@@ -1007,7 +1002,8 @@ impl StoreService {
 
         if let Some(data) = span.data {
             for (key, raw_value) in data {
-                let Some(json_value) = raw_value else {
+                let Some(json_value) = raw_value.and_then(|raw| Deserialize::deserialize(raw).ok())
```

Comment:
Logically it shouldn't be possible for this `deserialize` to fail because `raw_value` is a `RawValue` and so definitely valid JSON. Still, I would rather not `unwrap` here.

Reply:
We should probably add a `relay_log::error` here in the error case.